frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -420,10 +416,19 @@ public void doView(RenderRequest request, RenderResponse response) throws IOExce
     }
 
     private void prepareStandardView(RenderRequest request) throws IOException {
-
         String searchtext = request.getParameter(KEY_SEARCH_TEXT);
-
         String searchfilter = request.getParameter(KEY_SEARCH_FILTER_TEXT);
+        List<Project> projectList = getFilteredProjectList(request);
+
+        request.setAttribute(PROJECT_LIST, projectList);
+        request.setAttribute(KEY_SEARCH_TEXT, searchtext);
```

Comment:
It's true here, though, since the searchtext is set also in the getFilteredProjects() method. And the KEYWORD_SEARCH and therefore the searchfilter is ignored anyway when refreshing the page.